06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md

```diff
@@ -0,0 +1,2 @@
+###Output
+[Repository for Project](https://github.com/andela-tAdedotun/Inverted-Index)
```

Comment:
The Code Climate badge is displaying the health of the master branch which has extraneous files in it. Once I merge my open pull request, everything will be better.